IElementInternals missing recently added ariaDescription type field

TypeScript recently added `ariaDescription` as a type field in its automatically generated DOM typings in this [commit](https://github.com/microsoft/TypeScript/commit/d69757e3d39d48eaa798e6e542dd416f8aac2c93). 

Updating to TypeScript 5.4+ throws `error TS2741: Property 'ariaDescription' is missing in type 'ElementInternals & IElementInternals' but required in type 'ElementInternals'`

--- a/src/aom.ts
+++ b/src/aom.ts
@@ -11,6 +11,7 @@ export const aom: IAom = {
   ariaColIndexText: 'aria-colindextext',
   ariaColSpan: 'aria-colspan',
   ariaCurrent: 'aria-current',
+  ariaDescription: 'aria-description',
   ariaDisabled: 'aria-disabled',
   ariaExpanded: 'aria-expanded',
   ariaHasPopup: 'aria-haspopup',

--- a/src/element-internals.ts
+++ b/src/element-internals.ts
@@ -37,6 +37,7 @@ export class ElementInternals implements IElementInternals {
   ariaColIndexText: string;
   ariaColSpan: string;
   ariaCurrent: string;
+  ariaDescription: string;
   ariaDisabled: string;
   ariaExpanded: string;
   ariaHasPopup: string;

--- a/src/types.ts
+++ b/src/types.ts
@@ -11,6 +11,7 @@ export interface IAom {
   ariaColIndexText: string;
   ariaColSpan: string;
   ariaCurrent: string;
+  ariaDescription: string;
   ariaDisabled: string;
   ariaExpanded: string;
   ariaHasPopup: string;

